tilavarauspalvelu/migrations/0043_populate_uuid_values.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 5.1.3 on 2024-11-28 07:30
+from __future__ import annotations
+
+import uuid
+
+from django.db import migrations
+
+
+def gen_uuid(apps, schema_editor) -> None:
+    Reservation = apps.get_model("tilavarauspalvelu", "Reservation")
+    rows = list(Reservation.objects.all())
+    for row in rows:
+        row.ext_id = uuid.uuid4()
+    Reservation.objects.bulk_update(rows, ["ext_id"])
```

Comment:
This migration should be unnecessary, due to `models.UUIDField(default=uuid.uuid4, ...)` already handling it

Reply:
This is straight from [Django's own instructions](https://docs.djangoproject.com/en/5.1/howto/writing-migrations/#migrations-that-add-unique-fields) on how to add unique fields. The middle step is required to make sure that new values are actually unique.